src/amber/controller/helpers/tags.cr

```diff
@@ -1,5 +1,5 @@
 module Amber::Controller::Helpers
-  module Tag
+  module CSRF
```

Comment:
Probably name the file csrf.cr as well if you're changing the name of the module.